test/byrow.jl

```diff
@@ -99,13 +98,6 @@ end
     end
     @test df2 == DataFrame(A = [2, 1, 2], B = [2, 1, 2])
 
-    n = 1
-    df2 = @byrow df begin
-        cols(n) = :B
-    end
-    @test df2 == DataFrame(A = [2, 1, 2], B = [2, 1, 2])
```

Comment:
Isn't this still allowed?

Reply:
No, because of the change with `@with`. We now disallow mixing `:x` and `2` inside the inputs to the anonymous function. 

and `@byrow` is just 

`@with df begin 
    for row in 1:nrow(df)
        :x[row] = :y[row]
    end
end
```

The distinction between LHS and RHS isn't present in `@with` the way it is in `@transform`.